cmd/jobbuzz-api/main.go

```diff
@@ -1,15 +1,14 @@
 package main
 
 import (
+	"github.com/b-open/jobbuzz/internal/config"
 	"github.com/b-open/jobbuzz/pkg/controller"
 	"github.com/gin-gonic/gin"
 )
 
 func main() {
+	config.InitDb()
```

Comment:
Using the function without any return value feels weird for me.

Maybe use this pattern instead?

https://stackoverflow.com/a/60324692

Create a controller struct which has the db object, then all controller functions are the receiver of that controller struct.